doc/operator-capabilities.md

```diff
@@ -0,0 +1,193 @@
+# Operator Capability Levels
+
+Operators come in different maturity levels in regards to their lifecycle management capabilities for the application or workload they deliver. The capability models aims to provide guidance in terminology to express what features users can expect from an operator.
+
+![operator-capability-level](./images/operator-capability-level.png)
+
+Each capability level is associated with a certain set of management features the Operator offers around the managed workload. Operator that do not manage a workload and/or are delegating to off-clusters orchestration services would remain at Level 1. Capability levels are accumulating, i.e. Level 3 capabilities require all capabilities desired from Level 1 and 2.
+
+
+## Terminology
+
+**Operator** - the custom controller installed on a Kubernetes cluster
+
+**Operand** - the managed workload provided by the Operator as a service
+
+**Custom Resource (CR)** - an instance of the `CustomResourceDefinition` the Operator ships that represents the Operand or an Operation on an Operand
```

Comment:
WDYT about we add a link for k8s doc here?
See: https://kubernetes.io/docs/tasks/access-kubernetes-api/custom-resources/custom-resource-definitions/

Reply:
Do you propose to use the word API instead of CRD?

Reply:
No.